### ISSUE TITLE
When i do a composer update, i do not get the new version, is this normal?

### ISSUE THREAD

**Issue by j0k3r** [smalot/pdfparser#381]
Thanks @k00ni Just one question, sorry this is certainly a stupid question, but when i do a composer update, i do not get the new version, is this normal?

_Originally posted by @mathias1981 in https://github.com/smalot/pdfparser/issues/380#issuecomment-752468528_

**Comment by k00ni** [comment]
There are no stupid questions :)

I think there is a misunderstanding here. Latest version of PDFParser is still 0.17.1. But there are changes since which can be seen here: https://github.com/smalot/pdfparser/compare/v0.17.1...master. My fix for #367 is not part of the master (branch) yet, but will be after it has been reviewed and accepted. Then you will see it in that list also.

To be able to get this fix (and all other changes since 0.17.1) I have to create a new version manually. This I (or others) will do after #380 got merged. Hope it helps.
